src/main/java/jalau/at18/spaceinvaders/Bullet.java

```diff
@@ -0,0 +1,52 @@
+package jalau.at18.spaceinvaders;
```

Comment:
This class should be abstract to work on the movement of the ship's bullet and aliens separately